src/mlpack/core/cv/metrics/top_k_accuracy_impl.hpp

```diff
@@ -29,32 +30,31 @@ double TopKAccuracy::Evaluate(MLAlgorithm& model,
         << std::endl;
     throw std::invalid_argument(fpp.str());
   }
-  arma::Row<size_t> predictedLabels;
+  arma::Row<size_t> predictionProbs;
```

Comment:
This only works for a single sample, but not if I pass multiple samples, better use `DataType`. Also if you switch to `DataType` make sure to go over all samples.

Reply:
Yeah, you are right, making the changes in the commit. Thanks for suggesting

Reply:
@zoq So do I need to iterate over the prediction probabilities?

Reply:
@zoq can you tell me why the style showing the same error after a change?

Reply:
No looks like a false warning, unless somebody else can see what is wrong I'll go ahead an whitelist the issue.

Reply:
Hi, @zoq I exactly don't know what is wrong with the style, Can you please check if someone having similar problems in style. Thanks for your help.

Reply:
Hey @Shandilya21, I think we can ignore the style warning for now. It will be whitelisted if it persists as mentioned above. Maybe fixing the build should be the priority for now. Let me know if that makes sense.
Thanks a lot for working on this.

Reply:
Yeah, @kartikdutt18 I am working on that and made the changes accordingly. I have to do some updates in the `cv_test.cpp`, then I will push the code. Thanks for your comment.

Reply:
Great!
Looking forward to it.
Thanks.